src/parser/node/cond.rs

```diff
@@ -4,20 +4,21 @@
 //! rule
 //!
 //! ```abnf
-//! cond = "if" block1 "do" block1 ("else" block1)? "end"
+//! cond = "if" block1 "do" block1 ("elif" block1 "do" block1) ("else" block1)? "end"
```

Comment:
change `("elif" block1 "do" block1)` to  `("elif" block1 "do" block1)*` to indicate that there could be zero or more of those.

Also change the `("else" block1)?` by just `"else" block1`. That used to be the grammar when we had optional `else`s but we no longer do that and I forgot to update the docs :/